rfcs/formats/views/request.md

```diff
@@ -12,3 +12,84 @@
 
 ## Summary
 
+## Query
+
+Query message is used to define the data that is being requested. This is used in relay-relay communication, where destination relay requests response for this query from source relay.
+
+```protobuf
+message Query {
+  repeated string policy = 1;
+  string address = 2;
+  string requesting_relay = 3;
+  string requesting_network = 4;
+  string certificate = 5;
+  string requestor_signature = 6;
+  string nonce = 7;
+  string request_id = 8;
+  string requesting_org = 9;
+}
+```
+
+## State
+
+### ViewPayload
+
+Encapsulates either error or the response view, identified by `request_id`.
+
+```protobuf
+// View represents the response from a remote network
+message ViewPayload {
+  string request_id = 1;
+  oneof state {
+    View view = 2;
+    string error = 3;
+  };
+}
+```
+
+The `state` contains `view` field (if success), which is an instance of `View` defined [here](./view-definition).
```

Comment:
Say that it either consists of a view (if the view/data sharing request was successful), otherwise it consists of an error report.